src/NServiceBus.Transport.RabbitMQ/NServiceBus.Transport.RabbitMQ.csproj

```diff
@@ -8,7 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="Fody" Version="3.3.3" PrivateAssets="All" />
-    <PackageReference Include="NServiceBus" Version="[7.0.0, 8.0.0)" />
+    <PackageReference Include="NServiceBus" Version="[7.1.6, 8.0.0)" />
```

Comment:
Should this be `[7.1.0,8.0.0)`?

Reply:
Why? 7.1.0 was never released, so it would already need to be a patch version. Since that's the latest version out, I don't see a reason not to set the minimum version to it.